src/init.cpp

```diff
@@ -368,6 +386,16 @@ std::string HelpMessage(HelpMessageMode mode)
         " " + _("(1 = keep tx meta data e.g. account owner and payment request information, 2 = drop tx meta data)"));
 #endif
 
+#if ENABLE_ZMQ
+    strUsage += "\n" + _("ZeroMQ notification options:") + "\n";
+    strUsage += "  -zmqpubhashblock=<address>  " + _("Enable publish hash block in <address>") + "\n";
+    strUsage += "  -zmqpubhashtx=<address>     " + _("Enable publish hash transaction in <address>") + "\n";
+    strUsage += "  -zmqpubhashtxlock=<address> " + _("Enable publish hash transaction (locked via SwiftTX) in <address>") + "\n";
+    strUsage += "  -zmqpubrawblock=<address>   " + _("Enable publish raw block in <address>") + "\n";
+    strUsage += "  -zmqpubrawtx=<address>      " + _("Enable publish raw transaction in <address>") + "\n";
+    strUsage += "  -zmqpubrawtxlock=<address>  " + _("Enable publish raw transaction (locked via SwiftTX) in <address>") + "\n";
```

Comment:
Possible to reformat these help messages to conform to the new standard using `HelpMessageGroup` and `HelpMessageOpt` as exampled in the surrounding lines?